### PR TITLE
Cleaning up

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
- [x] pyproject
- [x] comments
- [x] move all bbox duties under bbox backend
- [x]  docstrings

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 django>=4
 django-computedfields
-psycopg2
+psycopg2-binary
```
Comment (why-not-try-calmer) [inline requirements.txt:3]: the `-binary` version comes precompiled.

File: pyproject.toml [hunk 2]
```diff
@@ -3,6 +3,7 @@ requires = ["setuptools>=45", "setuptools_scm[toml]==7.*"]
 build-backend = "setuptools.build_meta"
 
 [project]
+requires-python = ">3.9.6"
```
Comment (3nids) [inline pyproject.toml:6]: I thought we were on 3.7. What's the reasoning for this specific version?

File: src/signalo/core/views.py [hunk 3]
```diff
@@ -32,7 +32,6 @@ class PoleHighPerfViewset(OAPIFDescribeModelViewSetMixin, viewsets.ModelViewSet)
     pagination_class = HighPerfPagination
     oapif_title = "Poles (high perf)"
     oapif_description = "Poles layer - including high performance optimization"
-    # (one day this will be retrieved automatically from the serializer)
     oapif_geom_lookup = "geom"
```
Comment (3nids) [inline src/signalo/core/views.py:35]: this deserves a TODO, no?

File: pyproject.toml (hunk 2 above)
Comment (why-not-try-calmer) [reply pyproject.toml:6]: https://github.com/opengisch/django-ogcapif/blob/main/.github/workflows/publish.yml#L17 is the reason, but happy to change if you prefer

File: src/signalo/core/views.py (hunk 3 above)
Comment (why-not-try-calmer) [reply src/signalo/core/views.py:35]: on it

**Pull request merged**